lib/mojito.js

```diff
@@ -428,7 +428,7 @@ Mojito = (function ()
             chooseRecipe: function ()
             {
                 var recipes = this.options.recipes;
-                var chosen_recipe, partitions, chosen_partition, sameRecipes;
+                var chosenRecipe, partitions, chosenPartition, sameRecipes;
 
                 // check if sample property was set in recipe objects
                 var samplesWasSet = true;
```

Comment:
Just to tidy up the english used in our variables:

`samplesWasSet` >> `samplesSet`

Reply:
@dapperdrop 
Fixed.